resources/Components/RequestItem/index.js

```diff
@@ -40,7 +40,7 @@ class _RequestItem extends React.Component {
     let requestItemDetailsClass = 'requestItemDetails'
     let requestItemNoticeClass = 'requestItemNotice'
 
-    if (['sending', 'verifying', 'confirming', 'confirmed'].includes(req.status)) {
+    if (['sent', 'sending', 'verifying', 'confirming', 'confirmed'].includes(req.status)) {
```

Comment:
this may be another candidate for re-location, essentially moving all of these logic rules into one place so they can be easily standardized. unless this one is very specific to this component then of course it can just stay here

Reply:
I think we push on this one for now. There might be something here along the lines of introducing a kind of request meta status `good`, `bad` or `neutral` that is returned from the status but I'm not sure if that matches up with how other components do it so it would be isolated to this request for now -- will keep an eye towards combining and moving this logic in the future.